float64 probability
int64 xmin
int64 ymin
int64 xmax
int64 ymax
int64 class_no
bool is_green

